projects/angular/src/lib/mount.ts

```diff
@@ -85,11 +116,9 @@ export function mount<T extends object>(
 
   const fixture = setupFixture(component, testBed, autoDetectChanges);
 
-  testBed.compileComponents();
```

Comment:
This shouldn't be needed as it's implicitly done by TestBed anyway.
The only async thing is fetching resources (styles/templates) but this is done at build time... so it doesn't make much sense today.